### PR TITLE
Add documentation about how to deal with RHUI

### PR DESCRIPTION
**Pull request by meaksh**
This PR adds documentation about how to deal with RHUI repositories in order to allow syncing content using `spacewalk-repo-sync`.

@paususe have to decide if we add this or not to the documentation.

### CODE REVIEW

File: modules/client-configuration/pages/clients-rh.adoc [hunk 1]
```diff
@@ -109,6 +108,14 @@ cp /etc/rhsm/ca/redhat-uep.pem /example/entitlement
 To manage repositories on your {redhat} client, you need to import the CA and entitlement certificates to the {productname} Server.
 This requires three entries: one each for the entitlement certificate, the entitlement key, and the {redhat} certificate.
 
+[NOTE]
+====
+If your {redhat} client is using Red Hat Update Infrastructure (RHUI) repositories, you would need to use a different set CA and entitlement certificates. As an example:
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc:113]:
```suggestion
If your {redhat} client is using Red Hat Update Infrastructure (RHUI) repositories, you need to use a different set of CA and entitlement certificates. 
For example:
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 2]
```diff
@@ -202,6 +209,24 @@ Ensure you use the correct {rhela} version:
 . Click btn:[Sync Now] to begin synchronization immediately.
 
 
+[IMPORTANT]
+====
+In order to allow [command]``spacewalk-repo-sync`` to sync content from RHUI repositories, we need to manually add the required [systemitem]``X-RHUI-ID`` and [systemitem]``X-RHUI-SIGNATURE`` required HTTP headers to [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` file:
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc:214]:
```suggestion
For RHUI repositories, you must manually add the [systemitem]``X-RHUI-ID`` and [systemitem]``X-RHUI-SIGNATURE`` required HTTP headers to the [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` configuration file.
Without these headers, the [command]``spacewalk-repo-sync`` command will not be able to synchronize content from the RHUI repositories.
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 3]
```diff
@@ -202,6 +209,24 @@ Ensure you use the correct {rhela} version:
...
+
+----
+[channel_label]
+X-RHUI-ID=value
+X-RHUI-SIGNATURE=value
+----
+
+An example about how to calculate those values for the required HTTP headers from a {rhela} instance running on the Public Cloud:
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc:222]:
```suggestion
Use this example to calculate the the required values for the HTTP headers from a {rhel} instance running on a public cloud:
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 4]
```diff
@@ -228,6 +253,23 @@ Ensure you use the correct {rhela} version:
...
+
+----
+echo "X-RHUI-ID=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document|base64|tr -d '\n')"
+echo "X-RHUI-SIGNATURE=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/signature|base64|tr -d '\n')"
+----
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc:271]: See my edits above :point_up:

Comment (meaksh) [reply modules/client-configuration/pages/clients-rh.adoc:274]: Done. Thanks!

**Comment by Loquacity** [comment]
****DO NOT MERGE****

Waiting until @paususe gives the go-ahead.

**Comment by Loquacity** [comment]
@paususe are we OK to merge this before we package for beta 4?

**Comment by paususe** [comment]
> @paususe are we OK to merge this before we package for beta 4?

No

**Comment by Loquacity** [comment]
> > @paususe are we OK to merge this before we package for beta 4?
> 
> No

@paususe Last chance to get this in for beta 4 ...

**Comment by paususe** [comment]
NOK for now

**Comment by Loquacity** [comment]
This will need to be re-addressed. Created https://github.com/SUSE/spacewalk/issues/11539 so we don't lose it.

**Comment by Bischoff** [comment]
I tested the functionality and found a few gotchas with this PR. I'll list them in @Loquacity 's new card.

**Pull request closed without merging**